api/api.yaml

```diff
@@ -153,3 +153,36 @@ paths:
               schema:
                 type: string
                 example: invalid file format
+
+  /format:
+    post:
+      tags: ['Bai2 Files']
+      summary: Format bai2 file after parse bin file
+      description: format bai2 file.
+      operationId: format
+      requestBody:
+        content:
+          multipart/form-data:
+            schema:
+              properties:
+                input:
+                  type: string
+                  description: bai2 bin file
+                  format: binary
+            encoding:
+              file:
+                contentType: text/plain
+      responses:
+        '200':
+          description: successful operation
+          content:
+            application/json:
+              schema:
+                type: object
```

Comment:
This return type could be strongly typed, if preferred.

Reply:
Yea, can we reuse the OpenAPI models ?

Reply:
@adamdecaf sure thing.

I pushed two subsequent commits.

The first fixes the build with respect to lint, etc.

The second adds OpenAPI models for this endpoint. I didn't see any existing models and took a first pass here -- please let me know what changes you'd like to see from here.

_There seems to be an active GH outage that is currently these latest commits showing up (2024-03-12 7:45 PM CST) but I'm expecting a new build to trigger eventually._